js-java-async-helidon/pom.xml

```diff
@@ -38,24 +38,25 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
                 <configuration>
-                    <source>8</source>
-                    <target>8</target>
+                    <source>21</source>
+                    <target>21</target>
                 </configuration>
             </plugin>
         </plugins>
     </build>
 
     <properties>
-        <graalvm.version>21.2.0</graalvm.version>
+        <graalvm.version>21.0.1</graalvm.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>io.helidon</groupId>
                 <artifactId>helidon-bom</artifactId>
-                <version>1.4.4</version>
```

Comment:
@woess, upgrading this demo to Oracle GraalVM 21.0.1 requires (1) to import JS dependency in pom.xml and (2) to upgrade helidon-bom to 4x to be compatible with Java 21. 
The helidon-bom dependency doesn't resolve, project compilation fails. Could you please check https://github.com/graalvm/graalvm-demos/actions/runs/7129108229/job/19412450125?